### ISSUE TITLE
HTTP: Authz failure leads to response committed warnings in log

### ISSUE THREAD

**Issue by reinkrul** [nuts-foundation/nuts-node#1974]
When authentication using token_v2 fails, it logs the authz error (which is expected):

```
stable-stable-nuts-node-1               | time="2023-03-10T12:49:19+01:00" level=audit msg="Access denied: expected issuer (demo-apps) does not match iss" actor=172.20.0.11 event=AccessDenied module=http operation=tokenV2.middleware
stable-stable-nuts-node-1               | time="2023-03-10T12:49:19+01:00" level=info msg="HTTP request" method=GET module=http remote_ip=172.20.0.11 status=401 uri="/internal/didman/v1/did/did:nuts:7eaDstczhnigW9uW1JRzYF2u5KhTntFTnee8vGDHokD/contactinfo"
stable-stable-nuts-node-1               | time="2023-03-10T12:49:19+01:00" level=warning msg="Operation failed" error=Unauthorized operationID="<nil>" 
```

But then it logs a warning, which is unexpected:

```
stable-stable-nuts-node-1               | time="2023-03-10T12:49:19+01:00" level=warning msg="Unable to send error back to client, response already committed" error=Unauthorized
```

**Comment by reinkrul** [comment]
Really minor and might not be easy to fix, I'd say can be fixed after v5.1 release.

**Comment by woutslakhorst** [comment]
I also noticed that when unauthorized, there's a body containing the text `unauthorized`
